code/go/internal/validator/semantic/validate_kibana_no_dangling_object_ids.go

```diff
@@ -12,6 +12,7 @@ import (
 	ve "github.com/elastic/package-spec/v2/code/go/internal/errors"
 	"github.com/elastic/package-spec/v2/code/go/internal/fspath"
 	"github.com/elastic/package-spec/v2/code/go/internal/pkgpath"
+	"golang.org/x/exp/slices"
```

Comment:
Nit. Import order.

Reply:
I trusted the IDE xD 
I'll check it, sure!